src/status_im2/contexts/wallet/collectible/view.cljs

```diff
@@ -1,16 +1,110 @@
 (ns status-im2.contexts.wallet.collectible.view
   (:require [react-native.core :as rn]
             [quo2.core :as quo]
-            [re-frame.core :as rf]))
+            [utils.i18n :as i18n]
+            [status-im2.common.scroll-page.view :as scroll-page]
+            [status-im2.contexts.wallet.collectible.style :as style]
+            [status-im2.contexts.wallet.common.temp :as temp]))
 
 (defn view
   []
-  [rn/view
-   {:style {:flex            1
-            :align-items     :center
-            :justify-content :center}}
-
-   [quo/text {} "COLLECTIBLES PAGE"]
-   [quo/divider-label]
-   [quo/button {:on-press #(rf/dispatch [:navigate-back])}
-    "NAVIGATE BACK"]])
+  (let [{:keys [name description image collection-image traits]} temp/collectible-details]
+    [scroll-page/scroll-page
+     {:navigate-back? true
+      :height         148
+      :page-nav-props {:type        :title-description
+                       :title       name
+                       :description description
+                       :right-side  [{:icon-name :i/options
+                                      :on-press  #(js/alert "pressed")}]
+                       :picture     image}}
+     nil
+     [rn/view {:style style/container}
+
+      [rn/image
+       {:source image
+        :style  style/preview}]
+
+      [rn/view {:style style/header}
+       [quo/text
+        {:weight :semi-bold
+         :size   :heading-1} name]
+       [rn/view style/collection-container
+        [quo/collection-avatar {:image collection-image}]
+        [quo/text
+         {:weight :semi-bold
+          :size   :paragraph-1}
+         description]]]
+
+      [rn/view {:style style/buttons-container}
+       [quo/button
+        {:container-style style/send-button
+         :type            :outline
+         :size            40
+         :icon-left       :i/send}
+        (i18n/label :t/send)]
+       [quo/button
+        {:container-style style/opensea-button
+         :type            :outline
+         :size            40
+         :icon-left       :i/opensea}
+        (i18n/label :t/opensea)]]
+
+      [quo/tabs
+       {:size        32
+        :style       style/tabs
+        :scrollable? true
+        :data        [{:id                  :overview
+                       :label               (i18n/label :t/overview)
+                       :accessibility-label :overview-tab}
+                      {:id                  :activity
+                       :label               (i18n/label :t/activity)
+                       :accessibility-label :activity-tab}
+                      {:id                  :permissions
+                       :label               (i18n/label :t/permissions)
+                       :accessibility-label :permissions-tab}
+                      {:id                  :about
+                       :label               (i18n/label :t/about)
+                       :accessibility-label :about-tab}]}]
+
```

Comment:
EL, and there is a few more in this file

If component is too long, then would be good to break it up into smaller pieces